scripts/zones/Dynamis-Bastok/mobs/Vanguard_Defender.lua

```diff
@@ -0,0 +1,15 @@
+-----------------------------------
+-- Area: Dynamis Bastok
+--  NPC: Vanguard Defender
+-----------------------------------
+mixins = {require("scripts/mixins/job_special")};
+require("scripts/globals/status");
+
+function onMobSpawn(mob)
+end;
+
+function onMobEngaged(mob,target)
```

Comment:
*don't* need empty onMobEngaged or onMobDespawn functions in mob scripts.  can delete them.

*do* need to keep onMobDeath functions, even if empty.

Unsure about onMobSpawn, so I'd keep them.